portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path

```diff
@@ -175,6 +180,11 @@
 	<td>//input[contains(@name,'destinationURL_${key_index}')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
Please help check if we already have existing "PATH" to use and we should avoid adding the duplicate ones.

Reply:
Done! thanks.